src/System.Private.CoreLib/src/System/Diagnostics/Eventing/EventSource_CoreCLR.cs

```diff
@@ -136,95 +21,6 @@ private Type GetDataType(EventMetadata eventData, int parameterId)
             return eventData.Parameters[parameterId].ParameterType;
         }
 
-        private static string GetResourceString(string key, params object[] args)
-        {
-            return SR.Format(SR.GetResourceString(key), args);
-        }
-
         private static readonly bool m_EventSourcePreventRecursion = false;
```

Comment:
Is the stuff left in this file actually used for anything?

Reply:
It is. The CoreFX and CoreCLR implementation is identical. I can move it to the shared part under `!CORERT` if desired.

Reply:
I moved it to shared partition with the `!CORERT` guard. I honestly don't know why the CoreRT implementation is different and the Git history doesn't provide me with an answer. Since it's also used by ProjectN I hesitate to touch it.

Reply:
I figured that the shared code already guards the rest of the related code with `ES_BUILD_PN` so I moved the CoreRT code along as well.